cmd/snap-gpio-helper/cmd_export.go

```diff
@@ -0,0 +1,35 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"errors"
+)
+
+type cmdExportChardev struct {
+	ChipLabels []string `long:"chip-label" description:"source chip label(s) to match" required:"yes"`
```

Comment:
The spec mentions plurals of these:
```suggestion
	ChipLabels []string `long:"chip-labels" description:"source chip label(s) to match" required:"yes"`
```

Reply:
I thought it would be more readable to pass multiple `--chip-label` args instead of one large comma-separated entry `--chip-label label-1 --chip-label label-2` vs `--chip-labels label-1,label-2`.

After second thought, the latter looks more compact.